twisted/topfiles/8760.feature

```diff
@@ -0,0 +1 @@
+Client and server TLS connections now support ChaCha20 ciphers when available from OpenSSL.
```

Comment:
Yeah, we could do that. I wondered if we needed to talk about the endpoints as well. I may adjust to talk about those as well. =)